### PR TITLE
Rename discussion modal translations

### PR DESCRIPTION
**Pull request by wiwatsrt**

### CODE REVIEW

File: locale/core.yml [hunk 1]
```diff
@@ -225,10 +225,14 @@ core:
       delete_forever_button: => core.ref.delete_forever
       log_in_to_reply_button: Log In to Reply
       rename_button: Rename
-      rename_text: "Enter a new title for this discussion:"
       reply_button: => core.ref.reply
       restore_button: => core.ref.restore
 
+      # These translations are used by the rename discussion modal.
+      rename_modal:
```
Comment (dcsjapan) [inline locale/core.yml]: It's a good idea to separate these out, but I'd suggest "rename_discussion:" as the key.

(Please compare the keys for other modal discussions.)

Comment (wiwatsrt) [inline locale/core.yml]: It translations for fix error [#1083](https://github.com/flarum/core/pull/1083)

**Comment by franzliedke** [comment]
@wiwatsrt Thanks for your contribution!

@dcsjapan This was correct, or at least consistent with the keys in the code. I now [changed this](https://github.com/flarum/core/commit/57da4e24cb019aeaca8e2004cd4a83f60c4afed6) as you suggested - which also seems to be consistent with other modals. However, is this still correct? Right now, the key `rename_discussion` is nested under `discussion_controls`. Is that intended? I'd guess not, since - in theory - the modal is reusable in other contexts as well.

**Comment by dcsjapan** [comment]
@franzliedke Right, each of those keys now has one more namespace than it should.

Just remove the `discussion_controls.` from each key and it will match the resources in the YML.

**Comment by franzliedke** [comment]
@dcsjapan Note that the nesting in the YML file was not correct, either. I fixed both now.

**Comment by dcsjapan** [comment]
Thanks, didn't notice that it was out of order.

**Pull request merged**